components/Reservations/CreateReservationModal.tsx

```diff
@@ -0,0 +1,253 @@
+import { useState } from "react";
+import { addReservation } from "../../actions/calendarActions";
+
+export default function CreateReservation({ setIsShowing }) {
+    const [formData, setFormData] = useState({
+        reservationTime: "",
+        reservationDate: "",
+        walkDuration: "60",
+        name: "",
+        email: "",
+        phoneNumber: "",
+        city: "skopje",
+        municipality: "",
+        address: "",
+        dogBreed: "",
+        walkType: "socialization",
+    });
+
+    async function handleInputChange(e) {
+        const { name, value } = e.target;
+        setFormData((prevData) => ({
+            ...prevData,
+            [name]: value,
+        }));
+    }
+
+    async function submitForm(e) {
+        e.preventDefault();
+        const reservationDate = new Date(formData.reservationDate);
+
+        try {
+            const result = await addReservation(
+                reservationDate,
+                formData.reservationTime,
+                parseInt(formData.walkDuration, 10),
+                formData.name,
+                formData.email,
+                formData.phoneNumber,
+                formData.city,
+                formData.municipality,
+                formData.address,
+                formData.dogBreed,
+                formData.walkType
+            );
+
+            if (result.status === 200) {
+                alert(result.message);
```

Comment:
We don't need this alert, is slows down the workflow, we can see the results in the table instantly with the proposed modification bellow.

We can keep the error alert as that one actually needs to grab your attention.